lib/index.ts

```diff
@@ -25,3 +25,12 @@ export {
   ISidetreeBitcoinConfig,
   SidetreeBitcoinService
 };
+
+// IPFS service exports.
```

Comment:
Not sure if I like this "Super-Index". Sidetree-Bitcoin would not be interested in these dependencies. I think it would still be cleaner if the code would import individual sub-modules in `lib`.

Reply:
The hope here is that an external repo using this lib will be able to know what are the explicitly exported classes without needing to sift through the folders and being overwhelmed by classes, that are irrelevant (used internally). If you feel strongly about this, feel free to file an issue, and we can discuss this further.